storage/backend/boltoption_unix.go

```diff
@@ -30,5 +30,5 @@ import (
 // silently ignore this flag. Please update your kernel to prevent this.
 var boltOpenOptions = &bolt.Options{
 	MmapFlags:       syscall.MAP_POPULATE,
-	InitialMmapSize: InitialMmapSize,
+	InitialMmapSize: int(InitialMmapSize),
```

Comment:
bolt.Options expects an `int`; declaring `InitialMmapSize` as `int64` so that arm builds won't complain causes a type error without the `int` cast